homeassistant/components/synology_srm/device_tracker.py

```diff
@@ -132,7 +132,11 @@ def _update_info(self):
         """Check the router for connected devices."""
         _LOGGER.debug("Scanning for connected devices")
 
-        self.devices = self.client.core.network_nsm_device({"is_online": True})
+        try:
+            self.devices = self.client.core.get_network_nsm_device({"is_online": True})
+        except Exception as ex: # pylint: disable=broad-except
```

Comment:
Please catch the exception we expect. Don't catch broad exceptions.

Reply:
Done.